src/go/k8s/pkg/resources/configmap_test.go

```diff
@@ -104,7 +104,7 @@ func TestEnsureConfigMap_AdditionalConfig(t *testing.T) {
 			name:                    "Primitive object in additional configuration",
 			additionalConfiguration: map[string]string{"redpanda.transactional_id_expiration_ms": "25920000000"},
 			expectedStrings:         []string{"transactional_id_expiration_ms: 25920000000"},
-			expectedHash:            "13b7ea49c811062afe2a6d275604929e",
+			expectedHash:            "3700f26a4631d5ab31e148352fdea7e0",
```

Comment:
How difficult would this be to switch to a full golden comparison, rather than a hash on the file?

Reply:
I think it should be that hard to get the content of the config file and see the real value.

Reply:
https://github.com/redpanda-data/redpanda/pull/6938

Reply:
@RafalKorepta I just did the same, I'll revert my commit then. But let's merge this first